### DISCUSSION TITLE
Non-periodic boundary

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#2567] [Q&A]
Hello,

I hope you are doing well.  I'm familiar with how to train a model with non-periodic boundary conditions in one direction.

Is this needed if one wants to run a LAMMPS simulation with a fixed boundary condition in one direction, using DeePMD as the pair_style?

Thank you very much.  Just looking for clarification on this point.

**Comment by wanghan-iapcm** [comment]
It is not supported at the moment. One may work around to increase the vacuum layer in the direction of non-pbc.

**Reply by npiroozan** [reply]
I understand.  My apologies, but how would one increase the vacuum layer?

**Reply by njzjz** [reply]
Just give a large box, e.g. 999, in that direction.

When running LAMMPS simulations, the boundary condition is handled by LAMMPS.

**Reply by npiroozan** [reply]
Understood, I thought perhaps something needed to change with respect to the training phase. Thank you!
